ko/tutorials/tour/traits.md

```diff
@@ -41,4 +41,4 @@ language: ko
     true
     true
 
-윤창석, 이한욱 옮김
\ No newline at end of file
+윤창석, 이한욱 옮김
```

Comment:
Previous version did not have a newline character at the end.
I blame **vim** for automatically adding one.